hooks/ad-join-relation-joined.ps1

```diff
@@ -22,27 +22,28 @@ try {
     Import-Module JujuHooks
     Import-Module JujuUtils
 
+    $renameReboot = Rename-JujuUnit
```

Comment:
then we should add a ```BEGIN``` section in that commandlet and determine the version of juju. If we are running under ```2.x``` and the name is suitable, we should quit.